docs/source/azure/examples/sqldatabaselongtermbackupretention-databases.rst

```diff
@@ -1,7 +1,7 @@
 .. _azure_examples_sqldatabaselongtermbackupretention:
 
-Find SQL Databases with a Monthly Long Term Backup Retention More Than 1 Year
-=============================================================================
+SQL - Find SQL Databases with a monthly long term backup retention more than one year
```

Comment:
If these titles seem crazy they can easily be shortened as needed, e.g.

> SQL - Find databases with specific retention options

But if its fits/displays OK, then np.